source/PreferencesPanel.cpp

```diff
@@ -581,7 +586,11 @@ void PreferencesPanel::DrawControls()
 
 			zones.emplace_back(table.GetCenterPoint(), table.GetRowSize(), command);
 
-			table.Draw(command.Description(), medium);
+			const auto it = UNLOCKABLE_CONTROLS.find(command);
+			if(it != UNLOCKABLE_CONTROLS.end() && !GameData::GlobalConditions().Has(it->second))
```

Comment:
```suggestion
			if(UNLOCKABLE_CONTROLS.count(command) && !GameData::GlobalConditions().Has(it->second))
```

Reply:
no wait, why would I?

Reply:
Sorry, you're right. I missed where the iterator is used in the second part of the expression.